CONTRIBUTING.md

```diff
@@ -47,7 +47,7 @@ We welcome direct contributions to our smtpapi-go code base. Thank you!
 
 ##### Prerequisites #####
 
-- Go 1.6
+- Go 1.11-1.16
```

Comment:
This indicates we need _all_ the 1.11-1.16 versions installed :) 
Maybe call the section "Supported versions" instead?